### DISCUSSION TITLE
Issues completing the "tutorial" from a Windows machine

### DISCUSSION THREAD

**Discussion by Skewjo** [vercel/next.js#19029] [Help] [answered]
First off - I'm (very) new to web dev, and I apologize if this is a stupid question. 

I'm getting the following error after running "npm run dev" and attempting to load the webpage:
![image](https://user-images.githubusercontent.com/4118039/98731251-11a9b080-2363-11eb-9fe1-fa16016e83a8.png)

Step 1(npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter") appeared to have no installation issues and appears to have worked correctly. 

I've browsed through the following 2 links and have been unable to resolve the issue. 
https://github.com/vercel/next.js/issues/9022
https://fb.me/react-invalid-hook-call

Once again, apologies if this is a stupid question, but a friend recommended I use Next.js for a website I want to build and I want to stick to his recommendation. This issue seems awfully complex to be occurring during the intro, and I've spent a couple of hours now trying to get it figured out with no luck. Any help is greatly appreciated.

**Comment by lfades** [comment, marked as answer]
Can you try removing `node_modules` and installing them again, see if that works? Also any other details you can provide like your Node.js version can be useful here.

**Comment by Skewjo** [comment]
I was on Node version v12.18.3. Updating to v14.15.0 resolved the issue.
I probably should've known better, but thanks for your help nonetheless.
